Resources/config/services.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+    _defaults:
+        public: false
```

Comment:
Not needed as services as private by default since SF 4

Reply:
I know, I always put it explicitly as safety to avoid mis exposition by developers if they change the bundle.

Reply:
and btw it's from SF 3.4 ;)

Reply:
ok then :)